src/components/base-input-field/template.html

```diff
@@ -0,0 +1,10 @@
+<div class="inputField">
```

Comment:
Should we be using kebab-case for css class names as well? What is scss guideliness?